dask_expr/_collection.py

```diff
@@ -1900,8 +1895,13 @@ def __getattr__(self, key):
             return getattr(self.cat, key)
         elif key in self._dt_attributes:
             return getattr(self.dt, key)
-        elif key in self._monotonic_attributes:
-            return getattr(self, key)
```

Comment:
This is untested and goes into infinite recursion.
Needs follow-up as the property is faulty on Series too: TODO